src/test/java/subway/line/LineAcceptanceTest.java

```diff
@@ -0,0 +1,178 @@
+package subway.line;
+
+import org.apache.commons.lang3.tuple.Triple;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import subway.ApiTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static subway.line.LineSteps.*;
+import static subway.station.StationSteps.지하철역_생성;
+import static subway.station.StationSteps.지하철역_생성_요청;
+
+@DisplayName("지하철 노선 관련 기능")
+public class LineAcceptanceTest extends ApiTest {
+
+    /**
+     * Given: 지하철역을 2개 생성하고,
+     * Given: 새로운 지하철 노선 정보를 입력하고,
+     * When: 관리자가 노선을 생성하면,
+     * Then: 해당 노선이 생성되고 노선 목록에 포함된다.
+     */
+    @DisplayName("지하철 노선을 생성한다")
+    @Test
+    void createLine() {
+        // given
+        var 강남역 = 지하철역_생성(지하철역_생성_요청("강남역"));
+        var 강남역_ID = 강남역.jsonPath().getLong("id");
+
+        var 신논현역 = 지하철역_생성(지하철역_생성_요청("신논현역"));
+        var 신논현역_ID = 신논현역.jsonPath().getLong("id");
+
+        // given
+        var 신분당선_생성_요청 = 지하철_노선_생성_요청("신분당선", "bg-red-600", 강남역_ID, 신논현역_ID, 10);
+
+        // when
+        var 신분당선 = 지하철_노선_생성(신분당선_생성_요청);
+
+        // then
+        assertThat(신분당선.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        var 등록된_신분당선 = 신분당선.jsonPath().getObject("", LineResponse.class);
+        등록된_노선_정보_확인(등록된_신분당선, "신분당선", "bg-red-600", 강남역_ID, 신논현역_ID);
+    }
+
+    /**
+     * TODO
+     * - 지하철 노선을 생성할 때 상행역 또는 하행역 ID가 등록되지 않은 경우 요청이 실패한다
+     * - 지하철 노선을 생성할 때 상행역과 하행역이 같은 경우 요청이 실패한다
+     * - 지하철 노선을 생성할 때 거리가 0 이하인 경우 요청이 실패한다
+     * - 지하철 노선을 생성할 때 이름이 없거나 빈 문자열인 경우 요청이 실패한다
+     * - 지하철 노선을 생성할 때 색상이 없거나 빈 문자열인 경우 요청이 실패한다
+     */
+
+    /**
+     * Given: 지하철 노선이 여러 개 등록되어 있고,
+     * Given: 여러 개의 지하철 노선이 등록되어 있고,
+     * When: 관리자가 지하철 노선 목록을 조회하면,
+     * Then: 모든 지하철 노선 목록이 반환된다.
+     */
+    @DisplayName("지하철 노선 목록을 조회한다")
+    @Test
+    void showLines() {
+        // given
+        var 강남역 = 지하철역_생성(지하철역_생성_요청("강남역"));
+        var 강남역_ID = 강남역.jsonPath().getLong("id");
+        var 신논현역 = 지하철역_생성(지하철역_생성_요청("신논현역"));
+        var 신논현역_ID = 신논현역.jsonPath().getLong("id");
+        var 역삼역 = 지하철역_생성(지하철역_생성_요청("역삼역"));
+        var 역삼역_ID = 역삼역.jsonPath().getLong("id");
```

Comment:
> 제가 보통 도메인 및 서비스 단의 테스트 코드를 작성하는 편인데, 이번에 RestAssured을 사용하다보니 jsonPath()와 같은 메서드로 응답에 접근하다보니 내부의 값을 조회하기 위한 코드가 길어지면서 테스트 코드 자체의 가독성이 조금 떨어지는 경우도 있는 것 같습니다.
> 
> .jsonPath().getList(~~~~~).get~~~~ 
> 그래서 이런 부분에 대해서 리뷰어님께서는 어떻게 이슈를 해소하고 계신지 궁금합니다~🙂

질문에 대한 답을 남겨봅니다. 😊

```suggestion
        var 강남역 = 지하철역_생성(지하철역_생성_요청("강남역"));
        var 신논현역 = 지하철역_생성(지하철역_생성_요청("신논현역"));
        var 역삼역 = 지하철역_생성(지하철역_생성_요청("역삼역"));
```

저 같은 경우에는 ID 가지고오는 부분까지 API 요청 스텝에 모두 넣고 있습니다.(위 코드를 봤을 때 `지하철역_생성` 내부에 200 검증, id 가지고 오는 로직을 포함시키고 있습니다.) 

그 이유는 인수 테스트에서 사용하는 정보는 거의 한정적이라고 보기 때문이에요. 경험적으로 ID를 가지고 오는 경우가 훨씬 많더라구요. 단, ID 값이 아니라 다른 값들도 가지고 와야 하는 상황이라면, jsonPath()이나, 요청에 대한 응답 클래스를 반환하여 사용하기도 합니다. 


https://github.com/next-step/atdd-subway-map/pull/1198#discussion_r1668673397 <- 해당 리뷰와 내용이 비슷하네요 😊